ccd-definition/CaseField/CareSupervision/hearingOrders-nonprod.json

```diff
@@ -5,7 +5,8 @@
     "ID": "noDraftCMOs",
     "Label": "There are no draft orders to review.",
     "FieldType": "Label",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
```

Comment:
in general i would avoid having these elements in nonprod file. Most of them can be in hearingOrders.json. If not used can be uploaded in prod. but can stay as it is